WTWSMS/common/religions/00_religions.txt

```diff
@@ -1336,6 +1353,7 @@ muslim = {
 		uses_jizya_tax = no
 		can_hold_temples = no
 		defensive_attrition = yes #Yazidi originated deep in the mountains and hostile terrain
+		independence_war_score_bonus = 75

```

Comment:
I kept this Yazidi one because it represents just how hard to catch Yazidis (and Kurds) it is. See, the Yazidis survived every persecution ever made against them, and the Kurds, to this day, enjoy autonomy given not by a ruler, but by their terrain. Yazidis hide in the mountains and impassable terrain, where it is extremely difficult to stamp them out.

If Yazidis rise up and seize that impassable terrain, weeding them out and restoring order is a Herculean feat.